### DISCUSSION TITLE
High overdispersion with autopolyploid potato data

### DISCUSSION THREAD

**Discussion by heatherturtle** [lvclark/polyRAD#26] [Q&A]
I have over 450 individuals that are tetraploid and I used the GATK pipeline.  I have the vcf containing diploid calls that was filtered on missingness per individual, missingness per site, depth, FS, GQ, ReadPOsRankSum, and MQ.  When I follow the polyRad workflow, it says that overdispersion is optimal at 30.  However, after running ExpectedHindHe it leads me to results that remove over 95% of my markers (out of ~16K).  My thoughts are that I am filtering to stringently, but I would like your input on the results that I am getting. 

Thanks!

**Comment by lvclark** [comment]
Could you share your histogram of Hind/He values for markers with MAF > 0.05, prior to any filtering by Hind/He?

**Comment by heatherturtle** [comment]
Hi Lindsay,

Thanks for the quick response! Histogram is attached.

Best,
Heather

**Reply by lvclark** [reply]
Hi Heather,

I don't see the histogram.  Could you upload it directly on GitHub rather than attaching it to email?

Thanks,

Lindsay

**Reply by heatherturtle** [reply]
![HindHe](https://user-images.githubusercontent.com/53024233/154696527-7d4eb11b-ccc5-4542-a820-c0d657567ac5.jpeg)

**Reply by lvclark** [reply]
Ok, from this it looks like you should set inbreeding to about 0.3 when using `ExpectedHindHe`.  Is that the value that you used?

**Reply by heatherturtle** [reply]
That is not the value I used! It was set to zero, so it was a typo! I am only removing 30% of markers now. In an autopolyploid, I would only want to remove markers where Hind/He is too low, correct?

**Reply by lvclark** [reply]
I would recommend also removing markers where Hind/He is too high, even though you aren't working with an allopolyploid.  If Hind/He is too high, it could indicate misalignment or sequencing error.

Glad this helped!

**Comment by heatherturtle** [comment]
One more thing I want to be clear on, is that polyrad is intended to impute
tetraploid genotype calls from previously diploidized genotype calls,
is that correct?
Is it beneficial to use tetraploid calls (GATK is able to produce
tetraploid calls) before using polyRAD, or is it fine to keep them as
diploid calls?

Thank you for taking time out of your day to answer these questions, it is
greatly appreciated!

**Reply by lvclark** [reply]
That's actually not correct.  When polyRAD imports from VCF, it ignores the genotype calls entirely.  The allelic read depths from the VCF are used by polyRAD to make new genotype calls.

Still, I can see it being advantageous to do tetraploid genotype calling in GATK if you know your organism is tetraploid, because that may impact some of the filtering parameters.
